tests/src/mender-update/daemon/state_test.cpp

```diff
@@ -2895,13 +2930,13 @@ vector<StateTransitionsTestCase> GenerateStateTransitionsTestCases() {
 			.status_log =
 				{
 					"downloading",
-					// "installing", // Missing because of fail_status_report_status below
-					"rebooting",
+					"installing",
+					// "rebooting", // Missing because of fail_status_report_status below
 					"failure",
 				},
 			.install_outcome = InstallOutcome::SuccessfulRollback,
 			.fail_status_report_count = 100,
-			.fail_status_report_status = deployments::DeploymentStatus::Installing,
+			.fail_status_report_status = deployments::DeploymentStatus::Rebooting,
 			.fail_status_aborted = true,
 			// When aborting an update, it should react immediately.
 			.long_retry_times = true,
```

Comment:
What about test `Aborted_update_before_commit` ? I know that you have not changed the logic on this but I think is good to add the test.

Reply:
So there was a good reason for not being there in the first place :sweat_smile: I'll have a look but most likely I won't find a solution neither.